### PR TITLE
Fix IsIdentifiable service exception handling

### PR DESCRIPTION
**Pull request by rkm**
## Proposed Changes

-	Refactor `IsIdentifiableQueueConsumer`
	-	Improve exception handling to better handle errors caused by IClassifier
	-	Remove redundant `fileSystemRoot` from constructor
	-	Add tests

## Types of changes

What types of changes does your code introduce? Tick all that apply.

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation-Only Update (if none of the other choices apply)
  - In this case, ensure that the message of the head commit from the source branch is prefixed with `[skip ci]`

## Checklist

By opening this PR, I confirm that I have:

- [x] Reviewed the [contributing](https://github.com/SMI/SmiServices/blob/master/CONTRIBUTING.md) guidelines for this repository
- [x] Ensured that the PR branch is in sync with the target branch (i.e. it is automatically merge-able)
- [ ] Updated any relevant API documentation
- [x] Created or updated any tests if relevant
- [x] Created a [news](https://github.com/SMI/SmiServices/blob/master/news/README.md) file
    -   NOTE: This ***must*** include any changes to any of the following files: default.yaml, any of the RabbitMQ server configurations, GlobalOptions.cs
- [x] Listed myself in the [CONTRIBUTORS](https://github.com/SMI/SmiServices/blob/master/CONTRIBUTORS.md) file 🚀
- [x] Requested a review by one of the repository maintainers

## Issues

### CODE REVIEW

File: src/microservices/Microservices.IsIdentifiable/Service/IsIdentifiableQueueConsumer.cs [hunk 1]
```diff
@@ -1,75 +1,93 @@
...
             {
-                // Catch specific exceptions we are aware of, any uncaught will bubble up to the wrapper
-                ErrorAndNack(header, tag, "Error while processing AnonSuccessMessage", e);
+                ErrorAndNack(header, tag, $"Exception while classifying {message.GetType().Name}", ae);
                 return;
             }
 
-            _producer.SendMessage(new ExtractedFileVerificationMessage(message)
+            foreach (Failure f in failures)
+                Logger.Log(LogLevel.Info, $"Validation failed for {f.Resource} Problem Value:{f.ProblemValue}");
```
Comment (2bPro) [inline src/microservices/Microservices.IsIdentifiable/Service/IsIdentifiableQueueConsumer.cs:76]: Logger.Info?

Comment (rkm) [reply src/microservices/Microservices.IsIdentifiable/Service/IsIdentifiableQueueConsumer.cs]: Done!

**Review by 2bPro — COMMENTED** [review]
I think it makes sense. The changes to the consumer support easier testing and tests cover all likely cases. I checked if they ran ok on the CI/CD pipeline but coverage shows 0% for `Microservices.IsIdentifiable`. Is this as expected?

**Comment by rkm** [comment]
Thanks for the review! If that's all fine, could you click the approve button?

The coverage seems to be reporting correctly in the final table at the end of the test run?

**Review by 2bPro — APPROVED** [review]

**Pull request merged**